homeassistant/components/blebox/__init__.py

```diff
@@ -23,6 +23,7 @@
     Platform.AIR_QUALITY,
     Platform.LIGHT,
     Platform.CLIMATE,
+    Platform.BUTTON,
```

Comment:
Could you update this to be in alphabetical order?

Reply:
done